collector/rpc_metrics.go

```diff
@@ -142,25 +181,36 @@ func (collector *NodeRpcMetrics) Describe(ch chan<- *prometheus.Desc) {
 	ch <- collector.blockLagDesc
 	ch <- collector.blocksMissedDesc
 	ch <- collector.syncingDesc
+	ch <- collector.internalStatusRPCFail
+	ch <- collector.internalStatusRPCSuccess
+	ch <- collector.externalStatusRPCFail
+	ch <- collector.externalStatusRPCSuccess
 	ch <- collector.versionBuildDesc
 	ch <- collector.validatorRPCFail
 	ch <- collector.validatorRPCSuccess
 	ch <- collector.prevEpochKickoutDesc
 }
 
 func (collector *NodeRpcMetrics) Collect(ch chan<- prometheus.Metric) {
-	sr, err := collector.internalClient.Get("status", nil)
-	if err != nil {
-		ch <- prometheus.NewInvalidMetric(collector.versionBuildDesc, err)
-		return
-	}
+	fmt.Println("metrics collect")
```

Comment:
```suggestion
```
I'm also not opposed to keeping this, though wanted to point out in a review comment so you could delete more easily - assuming this was a debugging print

Reply:
```suggestion
	fmt.Println("Collecting Metrics...")
```
would be better imo if we did want to keep a permanent debugging statement here. Gonna be going to stdout anyways, so no harm

Reply:
Def agree